cmd/fleetctl/goquery.go

```diff
@@ -0,0 +1,132 @@
+package main
+
+import (
+	"fmt"
+	"strconv"
+
+	"github.com/AbGuthrie/goquery"
+	gqconfig "github.com/AbGuthrie/goquery/config"
+	gqhosts "github.com/AbGuthrie/goquery/hosts"
+	gqmodels "github.com/AbGuthrie/goquery/models"
+	"github.com/kolide/fleet/server/kolide"
+	"github.com/kolide/fleet/server/service"
+	"github.com/pkg/errors"
+	"github.com/urfave/cli"
+)
+
+type activeQuery struct {
+	status  string
+	results []map[string]string
+}
+
+type goqueryClient struct {
+	client       *service.Client
+	queryCounter int
+	queries      map[string]activeQuery
+	// goquery passes the UUID, while we need the hostname (or ID) to
```

Comment:
I don't know if this is totally right? goquery passes whatever you handed it. So you could hand it a hostname+query id parameter, and it'll pass them back

Reply:
As of now goquery uses the UUID of the returned host object to refer to the host later. I discussed with those folks the possibility of goquery using the original identifier, but that has not yet been implemented.

Reply:
Yes, it's reused. But, IIRC, it can be whatever you want it to be. A ulid is just a string